production/helm/loki/templates/admin-api/deployment-admin-api.yaml

```
@@ -97,14 +97,8 @@ spec:
             - name: http-memberlist
               containerPort: 7946
               protocol: TCP
-          {{- with .Values.adminApi.livenessProbe }}
-          livenessProbe:
-            {{- toYaml . | nindent 12 }}
-          {{- end }}
-          {{- with .Values.adminApi.readinessProbe }}
-          readinessProbe:
-            {{- toYaml . | nindent 12 }}
-          {{- end }}
+          {{- include "loki.adminApi.readinessProbe" . | nindent 10 }}
+          {{- include "loki.adminApi.livenessProbe" . | nindent 10 }}
           {{- with .Values.adminApi.startupProbe }}
           startupProbe:
```

For the completeness, please include `startupProbe` as well.

```suggestion
          {{- with .Values.adminApi.startupProbe | default .Values.loki.startupProbe }}
          startupProbe:
```